batchflow/models/validator/README.md

```diff
@@ -0,0 +1,59 @@
+# Validator
+
+Validator is an instrument to unify the process of model training and validation. To make model API, you need:
```

Comment:
Better structure is needed here.

Start with describing the reason, then provide a very basic usage example, then more advanced examples, then how to write all the classes and methods.